festflow/models.py

```diff
@@ -115,3 +117,29 @@ def __str__(self):
     class Meta:
         verbose_name = "about"
         verbose_name_plural = "about"
+
+class NewsLetter(models.Model):
+    """Stores newsletters sent to subscribed users
+    """
+    identifier = models.CharField(max_length=50, unique=True)
```

Comment:
I have a suggestion. How about we put  default=str(datetime.now())
That way users can append anything to it but it will remain unique.
What do you say?